docker-compose.yml

```diff
@@ -253,6 +254,9 @@ services:
       redisHost: "identity-service-redis"
       redisPort: 6379
 
+      aaoEndpoint: "http://anti-abuse-oracle:8000"
+      aaoAddress: "${AAO_WALLET_ADDRESS}"
```

Comment:
i see `AAO_WALLET` in config.json and `AAO_WALLET_ADDRESS` here. is that ok?

Reply:
`AAO_WALLET_ADDRESS` (address) is automatically generated from `AAO_WALLET` (private key)